Config/RDB Info.txt

```diff
@@ -1,336 +1,336 @@
-﻿########################
```

Comment:
no idea what you did with this file, but this file is already in CRLF & UTF-8 format in the current source.

seem you need to fix your repository.